src/contract.rs

```diff
@@ -142,6 +143,29 @@ impl<T: Transport> Instance<T> {
     {
         Ok(self.method(name, params)?.view())
     }
+
+    /// Returns a log stream for the specified event that emits a new log for
+    /// every new event emitted after the stream was created.
+    pub fn event<S>(&self, name: S) -> AbiResult<LogStream<T>>
+    where
+        S: AsRef<str>,
+    {
+        let event = self.abi.event(name.as_ref())?;
+        if event.anonymous {
+            todo!("anonymous events are currently not supported");
```

Comment:
til about anonymous events: https://ethereum.stackexchange.com/questions/73087/how-can-i-send-an-anonymous-event-in-abi-anonymous-true-in-solidity